crates/sui-benchmark/tests/simtest.rs

```diff
@@ -610,6 +611,12 @@ mod test {
         default_epoch_duration_ms: u64,
     ) -> TestCluster {
         init_test_cluster_builder(default_num_validators, default_epoch_duration_ms)
+            .with_authority_overload_config(AuthorityOverloadConfig {
+                check_system_overload_at_execution: false,
+                check_system_overload_at_signing: false,
```

Comment:
Maybe add some comments on why we get them disabled? I guess git blame can tell but some info will help for the future reader